v3/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "codap3",
-  "version": "3.0.0-mvp.1",
+  "version": "3.0.0-1085",
```

Comment:
I think we should use a version number that makes it clear that this is not an official release -- something like `3.0.0-pre.1085` or `3.0.0-alpha.1085`.